sklearn/metrics/tests/test_score_objects.py

```diff
@@ -247,6 +247,31 @@ def test_regression_scorers():
     assert_almost_equal(score1, score2)
 
 
+def test_log_loss():
+   
+
+
+
+    X = [[1,1], [1,1], [2, 2], [2, 2]]
+    y = [1, 1, 2, 2]
+    clf = DecisionTreeClassifier()
+    clf.fit(X, y)
+
+
+
+    y_score = clf.predict_proba([[2,2], [2,2]])
```

Comment:
I would make the probabilites in (0, 1) so that we are spared the eps handling in testing.